fission/src/util/TypeConversions.ts

```diff
@@ -72,10 +72,18 @@ export function MirabufVector3_JoltVec3(v: mirabuf.Vector3): Jolt.Vec3 {
     return new JOLT.Vec3(v.x / 100.0, v.y / 100.0, v.z / 100.0)
 }
 
+export function MirabufVector3_JoltFloat3(v: mirabuf.Vector3): Jolt.Float3 {
+    return new JOLT.Float3(v.x / 100.0, v.y / 100.0, v.z / 100.0)
+}
+
 export function MirabufFloatArr_JoltVec3(v: number[], offsetIndex: number): Jolt.Vec3 {
     return new JOLT.Vec3(v[offsetIndex] / 100.0, v[offsetIndex + 1] / 100.0, v[offsetIndex + 2] / 100.0)
 }
 
+export function MirabufFloatArr_JoltFloat3(v: number[], offsetIndex: number): Jolt.Float3 {
+    return new JOLT.Float3(v[offsetIndex] / 100.0, v[offsetIndex + 1] / 100.0, v[offsetIndex + 2] / 100.0)
+}
+
```

Comment:
Not that this is necessarily specific to this PR but what was the reasoning for this naming convention?

Reply:
The underscore just made it clearer in my mind